constellations/gemini-admin/src/ask_for_help/afhEdit.js

```diff
@@ -0,0 +1,234 @@
+"use strict";
+require("./afhEdit.sass");
+import React from "react";
+import moment from "moment";
+import API from "../api.js";
+import { AFHEditDateTime } from "./afhEditDateTime.js";
+import { Modal } from "../modals/modal.js";
+import { OkayModal } from "../modals/okayModal.js";
+import { YesNoModal } from "../modals/yesnoModal.js";
+
+export class AskForHelpEditPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isEdit: false,
+            id: 0,
+            date: null,
+            locationId: "",
+            notes: "",
+            subject: "",
+            locations: [],
```

Comment:
I think you forgot to initialize timeString and title, so I've been seeing errors in the Inspect Console.

But I think it would also help to re-order them as how you would displaying them.
```
id, date, timeString, subject, title, locationId, notes
```

And that way, you don't miss / forget any fields